src/main/java/nl/smallproject/www/techiteasy/dtos/CiModule/CiModuleInputDto.java

```diff
@@ -0,0 +1,44 @@
+package nl.smallproject.www.techiteasy.dtos.CiModule;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.PositiveOrZero;
+import jakarta.validation.constraints.Size;
+
+public class CiModuleInputDto {
+
+    @NotBlank(message = "{cimodule.name.notblank}")
+    @Size(min = 3, max = 255, message = "{cimodule.name.size}")
+    private String name;
+
+    @NotBlank(message = "{cimodule.type.notblank}")
+    @Size(min = 3, max = 255, message = "{cimodule.type.size}")
+    private String type;
+
+    @PositiveOrZero(message = "{cimodule.price.positiveornotzero}")
+    private double price;
+
+    public @NotBlank(message = "{cimodule.name.notblank}") @Size(min = 3, max = 255, message = "{cimodule.name.size}") String getName() {
```

Comment:
Je hoeft de validatie annotatie niet op de field EN op de getter te doen. Alleen op de field is voldoende